### PR TITLE
BIG cafes

### PR DESCRIPTION
**Pull request by asutula**
- Displays the three big cafes as only options
- Removes recommended cafes code
- Gets add custom cafe working

### CODE REVIEW

File: App/Components/CafesList.tsx [hunk 1]
```diff
@@ -1,97 +1,34 @@
...
 }
 
-interface StateProps {
-  recommended?: DiscoveredCafes
-}
-
-interface DispatchProps {
-  requestRecommendedCafes: () => void
-}
+const cafesBase64 = Config.RN_TEXTILE_CAFES_JSON
```
Comment (asutula) [inline App/Components/CafesList.tsx]: This value is read from the CircleCI "context" (both beta and production). It is a base64 encoded version of a json data structure that is represented by `Cafe[]`. I've stored the current valid json in 1Password as `production-cafes.json`

Comment (sanderpick) [reply App/Components/CafesList.tsx]: Sweet. Does the beta app just have one value then?

Comment (asutula) [reply App/Components/CafesList.tsx]: Currently, it's the same for prod and beta and you can manually enter the info for beta. But, I should make them different yea?

Comment (sanderpick) [reply App/Components/CafesList.tsx]: I think so yeah

Comment (asutula) [reply App/Components/CafesList.tsx]: Ok doing that

Comment (asutula) [reply App/Components/CafesList.tsx]: Ok the beta release will list dev and beta cafes

**Comment by asutula** [comment]
Ok g2g here i think

**Pull request merged**